api/v1alpha1/edgedevice_types.go

```diff
@@ -62,6 +65,14 @@ type HardwareProfileConfiguration struct {
 	Scope string `json:"scope,omitempty"`
 }
 
+type StorageConfiguration struct {
+	BucketHost         string `json:"bucket_host,omitempty"`
```

Comment:
Please use camelCase in json property names for consistency.

Reply:
there is no consistency in the file :-)